conf/ssl4eo-l-mocov2.yaml

```diff
@@ -0,0 +1,38 @@
+module:
+  _target_: torchgeo.trainers.MoCoTask
+  model: resnet50
+  weights: True
+  in_channels: 9
+  version: 2
+  layers: 2
+  hidden_dim: 2048
+  output_dim: 128
+  lr: 0.03
+  weight_decay: 1e-4
+  momentum: 0.9
+  schedule: [120, 160]
+  temperature: 0.07
+  memory_bank_size: 65536
+  moco_momentum: 0.999
+  gather_distributed: True
+  size: 224
+  grayscale_weights: null
```

Comment:
You shouldn't need to override any of these if you just want to use the defaults. They aren't required anymore like they were with the other trainers. I'll update the other trainers before 0.5.